decidim-admin/app/commands/decidim/admin/reorder_participatory_process_steps.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    # A command that sets a step in a participatory process as active (and
+    # unsets a previous active step)
+    class ReorderParticipatoryProcessSteps < Rectify::Command
+      # Public: Initializes the command.
+      #
+      # collection - an ActiveRecord::Relation of steps
+      # order_string - a String representing an Array holding the order of IDs
```

Comment:
I was confused because the var name says order_stirng, but then it's an Array of IDs, I didn't get it was a JSON string or similar (maybe it could be added)